docker-compose/compose.test.yml

```diff
@@ -15,50 +11,17 @@ services:
       CYPRESS_AAS_DISCOVERY_API_URL: 'http://mnestix-api:5064/discovery'
       ELECTRON_ENABLE_LOGGING: 1
     volumes:
-      - ../cypress-artifacts/screenshots:/cypress_Tests/cypress/screenshots
-      - ../cypress-artifacts/videos:/cypress_Tests/cypress/videos
+      - ./cypress-artifacts/screenshots:/cypress_Tests/cypress/screenshots
+      - ./cypress-artifacts/videos:/cypress_Tests/cypress/videos
     depends_on:
       - mnestix-browser
       - mnestix-api
-      - aas
+      - aas-environment
     networks:
       - mnestix-network
 
   mnestix-browser:
```

Comment:
In the pipeline the project is build right before the tests.
This will overwrite the local registry, so the built image is beeing used.
You could instead use the --build flag as well.

I would not hardcode the reset image here.